src/Controllers/LoginController.php

```diff
@@ -101,8 +101,12 @@ protected function getValidationRules(): array
      */
     public function logoutAction(): RedirectResponse
     {
+        // Capture logout redirect URL before auth logout,
+        // otherwise 'Call to a member function inGroup() on null' is thrown
```

Comment:
How about `hasPermission`. I mean, this text could have been written completely to cover everything.